src/orquestra/sdk/_base/_dsl.py

```diff
@@ -480,6 +486,40 @@ def __call__(self, *args: _P.args, **kwargs: _P.kwargs) -> _R:
             ),
         )
 
+    def resolve_task_source_data(
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Should this be private?